### DISCUSSION TITLE
raster-resampling nearest bug on windows fix w/o backporting

### DISCUSSION THREAD

**Discussion by laurens-in** [maplibre/maplibre-gl-js#3451] [General]
there's a bug where `raster-resampling: nearest` doesn't work in browsers on windows. the same bug existed in `mapbox-gl` and was fixed in this pull request: https://github.com/mapbox/mapbox-gl-js/pull/12577/files

I would love to fix this in `maplibre-gl` but I'm not sure how to do it without _backporting_.

**Comment by HarelM** [comment]
You need to fix it without looking at the mapbox code - i.e. debug, write some code, test etc. Ping me on slack of you need more clarifications.

**Reply by laurens-in** [reply]
I'm afraid I can't do that. I stumbled upon the issue in the `mapbox-gl` repo without even realizing. looked at the code to see what has to change exactly.  Since I'm not at all familiar with `mapbox`/`maplibre` internals I have no understanding of what's happening at all with the fix in the `mapbox` pull request.

**Reply by laurens-in** [reply]
is it possible to ask for permission to backport?

**Reply by HarelM** [reply]
I don't know, I never asked.
Feel free to ask :-)
